### PR TITLE
Add identity_id to hook payload

### PR DESCRIPTION
**Pull request by KaitCrawford**
Send the id for the identity that has opted out.

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@hodgestar @imsickofmaps small PR for review.
This is almost exactly the same as the previous one. If there are any other identity fields that you think subscribed applications might need please let me know.

**Comment by KaitCrawford** [comment]
Should I add version perhaps?

File: identities/models.py [hunk 1]
```diff
@@ -183,7 +183,8 @@ def handle_optout(sender, instance, created, **kwargs):
         sender=None,
         event_name='optout.requested',
         payload={
-            'details': identity.details,
+            'identity_id': str(identity.id),
```
Comment (hodgestar) [inline identities/models.py]: How come we need to convert the `id` into a string? What is it at the moment?

**Comment by hodgestar** [comment]
Nice catch. Other than the one small question, looks good.

File: identities/models.py (hunk 1 above)
Comment (imsickofmaps) [inline identities/models.py]: it's a UUID object, they aren't JSON encodable into celery tasks

Comment (imsickofmaps) [inline identities/models.py]: elsewhere in the system, we've just used `identity` to refer to the ID. I'd prefer that for consistency.

File: identities/models.py [hunk 2]
```diff
@@ -183,7 +183,8 @@ def handle_optout(sender, instance, created, **kwargs):
...
+            'identity_details': identity.details,
```
Comment (imsickofmaps) [inline identities/models.py:187]: the prefix makes more sense here given it's clarifying it's not the details of the optout

File: identities/models.py (hunk 1 above)
Comment (hodgestar) [inline identities/models.py]: @KaitCrawford Consider my portion of this resolved. :)

**Comment by KaitCrawford** [comment]
@hodgestar @imsickofmaps feedback addressed. ready for re-review

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**